README.md

```diff
@@ -57,6 +58,29 @@ Steerage.init({ config: Path.join(__dirname, 'config', 'config.json'), onconfig
 });
 ```
 
+### Example userConfigPaths usage
+
+Pass an array of fully resolved configuration document paths to `userConfigPaths` to merge each configuration into another. Configuration objects will be merged into the base configuration object initialized via the `config` option. Duplicate keys in configuration files will be overwritten upon merging. The values of the config file that is the first index of `userConfigPaths` takes precedence when merging.
+
+```javascript
+const Path = require('path');
+const Steerage = require('@vrbo/steerage');
+const Determination = require('@vrbo/determination');
+
+
+Steerage.init(
+    {
+        config: Path.join(__dirname, 'base', 'config.json'),
+        userConfigPaths: [
+            Path.join(__dirname, '..', 'final/config.json'), // Values of this config take precedence
+            Path.join(__dirname, '..', 'secondary/config.json')
```

Comment:
Seems like `config` and `userConfigPaths` should be one. But, backward compatible, so `config` could be a string or an array.

Reply:
👍 agreed, especially since an array is ordered and merging could just occur in order declared.

Reply:
I feel like I tried that approach and there was some issue resolving the base configs. That was several iterations ago so I'll revisit and see if I can consolidate the two.

Reply:
Also wondering how the on-config hook isn't enough to achieve the desired outcome? Typically the way I'd imagine layering in configuration would be:

steerage + user config -> on-config hook to merge configs (such as base configurations)

Reply:
Thanks for the suggestion I'll keep that in mind

Reply:
I was able to re-work things to merge the base configs within the onConfig hook, but it overwrites any user defined configs which is what I was worried about 🤔

Reply:
I did have an idea to create a custom shortstop handler called `execBase` then set that up with the baseDir for the base config of catalyst-server so that it resolves correctly regardless of if baseDir is explicitly set by the user. Seems to have solved the problem.